### PR TITLE
Turning in MP4

### PR DESCRIPTION
**Pull request by GGRice**
A fun game of pong!

### CODE REVIEW

**Comment by GGRice** [comment]
My updated version with music and more scores for MP5

File: pong.py [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+        """
+            Determines which player scores depending on if hit top or bottom
+        """
+        if self.y <= 0:
+            return 1
+        elif self.y >= 480:
+            return 2
+        else:
+            return 0
+
```
Comment (SungwooPark) [inline pong.py:188]: Nice organization of code. Clear and very readable.

**Comment by SungwooPark** [comment]
Nicely organized code, good documentation, and good use of MVC model. Good work!

**Comment by SungwooPark** [comment]
One more thing, when I actually try to run your program, it fails because it cannot find a music file that you are using.

**Pull request closed without merging**